napari/settings/_fields.py

```diff
@@ -30,7 +30,7 @@ def validate(cls, v):
             raise ValueError(trans._('must be a string', deferred=True))
 
         value = v.lower()
-        themes = available_themes()
+        themes = [str(t) for t in available_themes()]
```

Comment:
why this is required? `available_themes()` return tuple of strings. If list is required then it should be `list(available_themes())`